cloudinit/distros/gentoo.py

```diff
@@ -17,10 +17,16 @@
 
 
 class Distro(distros.Distro):
-    locale_conf_fn = "/etc/locale.gen"
+    locale_conf_fn = "/etc/env.d/02locale"
+    locale_gen_file = "/etc/locale.gen"
```

Comment:
being pedantic, for consistency's sake the new var should probably be suffixed with `_fn`, but I don't care too much.

Reply:
Thanks!